app/core/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+
+class Ingredient(models.Model):
+    """Ingredient to be used in a recipe"""
+    name = models.CharField(max_length=255)
+    recipe = models.ForeignKey(
+        'Recipe',
+        related_name="ingredients",
+        on_delete=models.CASCADE,
+        null=True
```

Comment:
`null=True` is not a good option here, since we wouldn't want orphan ingredients without recipes.